# Data path. adjust them!
common:
    image_path:  "/home/lnt/Datasets/fusion/multi_cali/image/"
    pcd_path:  "/home/lnt/Datasets/fusion/multi_cali/pcd/"
    result_path:  "/home/lnt/Datasets/fusion/multi_cali/extrinsic.txt"
    data_num: 5

camera:
    camera_matrix: [645.252100829746, 0.0,      646.8185880049397,
                0.0,     642.9685575891763,  353.62380523406574,
                0.0,     0.0,      1.0     ]
    dist_coeffs: [-0.051089818530300236, 0.03547486314704304, 0.0009051276804264704, 0.0012607789620934912, 0.000000]

# Calibration Parameters.!
calib:
    calib_config_file: "/home/lnt/catkin_ws/src/livox_camera_calib/config/config_indoor.yaml"
    use_rough_calib: true # set true if your initial_extrinsic is bad
